app/src/main/java/org/grapheneos/apps/client/ui/container/MainActivity.kt

```diff
@@ -135,6 +136,15 @@ class MainActivity : AppCompatActivity() {
         views.searchInput.addTextChangedListener { editable ->
             searchState.updateQuery(editable?.trim()?.toString() ?: "")
         }
+        views.searchInput.setOnEditorActionListener { input, actionId, _ ->
+            when (actionId) {
+                EditorInfo.IME_ACTION_SEARCH -> {
```

Comment:
i don't think this is required, `addTextChangedListener` is submitting query in realtime.

Reply:
see issue comment :)